guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -175,12 +204,96 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
     renderProperties()
     {
         var className = this.state.currentNode?.guiControl.getClassName();
+        if (className === "TextBlock") {

```

Comment:
A switch would be much more efficient in this case (especially since it can't two classes at the same time)

Reply:
There we go :)